lib/foundation.js

```diff
@@ -23,52 +25,60 @@ var palette = {
     normalActive: "#cad5df",
   },
   ink: {
-    lighter: "#bac7d5",
-    lighterHover: "#a6b6c8",
-    lighterActive: "#94a8be",
+    lighter: "#5F738C",
```

Comment:
Please replace for `light`, not for `lighter` shades 🙏 
`lighter` colors stay with the original values.